packages/stack/proxy/src/index.ts

```diff
@@ -1,9 +1,10 @@
-import { Embark, EmbarkEvents } from "embark-core";
+import { Embark, EmbarkEvents, EmbarkConfig } from "embark-core";
 import { __ } from "embark-i18n";
 import { Logger } from "embark-logger";
 import { buildUrl, findNextPort } from "embark-utils";
 
 import { Proxy } from "./proxy";
+import {hasOwnProperty} from "tslint/lib/utils";
```

Comment:
The `tslinst` package [is a devDependency](https://github.com/embarklabs/embark/blob/feat/ganache-plugin/packages/stack/proxy/package.json#L68) but it seems that here we're expecting it to be available in a production install.

Also, it seems the `hasOwnProperty` isn't used anywhere in this module. Do we actually need it?

Reply:
I looked in `<root>/node_modules/tslint/lib/utils.js` and the implementation of `hasOwnProperty` is very simple:

```js
function hasOwnProperty(arg, key) {
    return Object.prototype.hasOwnProperty.call(arg, key);
}
exports.hasOwnProperty = hasOwnProperty;
```

So, if we do need it, I recommend we inline it instead of importing it.

Reply:
Oops, you're right. I `hasOwnProperty` in a previous commit, but ended up not needing it. It seems like my editor import it from that file, but it shouldn't have needed it, because `Object` has that function already. 
Anyway, I removed it.